common/eth2/src/lighthouse_vc/types.rs

```diff
@@ -168,3 +168,8 @@ pub struct SingleExportKeystoresResponse {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub validating_keystore_password: Option<ZeroizeString>,
 }
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct SetGraffitiRequest {
+    pub graffiti: Option<GraffitiString>,
```

Comment:
Does this need to be an option? I think this would serialize to `null` if it's empty. If user wants an empty Graffiti, it should be an empty string.